python-frontend/src/main/java/org/sonar/python/tests/PytestUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.tests;
+
+import java.util.Set;
+import org.sonar.plugins.python.api.symbols.Symbol;
+import org.sonar.plugins.python.api.tree.QualifiedExpression;
+import org.sonar.python.tree.TreeUtils;
+
+public class PytestUtils {
```

Comment:
I tend to think this PytestUtils is not really needed, given that pytest is generally much more simple than unittest. We can probably check for the patterns/fully qualified names in the rules themselves.

Reply:
I agree with you, especially with the fact that I already have a dedicated method `isPytestRaise` for this.
I removed this class.